src/bicep/examples/iaas-dns-forwarders/README.md

```diff
@@ -0,0 +1,78 @@
+# Azure IaaS DNS Forwarders example
```

Comment:
Please append an entry to this in the root examples folder README.md table located: 
https://github.com/Azure/missionlz/tree/main/src/bicep/examples#example-explanations

Reply:
I've added an entry for this example to the root examples folder README.md table.